frontend/src/Components/TermsOfService.js

```diff
@@ -0,0 +1,29 @@
+import React, { useReducer } from 'react';
+import { reducer, initialState } from '../reducer';
```

Comment:
I don't think we're using our reducer here so you should be able to get away with just doing:
import React from "react";